.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
```

Comment:
The project should also be built with the oldest supported Python version--3.7. Though using a newer version for building may not cause an issue, using the oldest version will better ensure that the release is compatible with all supported versions.

Reply:
I corrected this to display the right version of python! I'll keep an eye out for things like this in the future.